### ISSUE TITLE
Fix security report

### ISSUE THREAD

**Issue by clemens-tolboom** [KennisnetwerkDataScience/Waterlaboratorium-Noord-IoT#10]
Github report on https://github.com/KennisnetwerkDataScience/Waterlaboratorium-Noord-IoT/network/alerts the issue below. After fixing this will disappear.

---

Remediation
Upgrade notebook to version 5.7.8 or later. For example:

[packages]

notebook = ">=5.7.8"
or…
[dev-packages]

notebook = ">=5.7.8"
Always verify the validity and compatibility of suggestions with your codebase.

Details
CVE-2019-10255 More information
moderate severity
Vulnerable versions: < 5.7.8
Patched version: 5.7.8
An Open Redirect vulnerability for all browsers in Jupyter Notebook before 5.7.8 and some browsers (Chrome, Firefox) in JupyterHub before 0.9.6 allows crafted links to the login page, which will redirect to a malicious site after successful login. Servers running on a base_url prefix are not affected.

**Comment by clemens-tolboom** [comment]
```bash
cd Team1

git pull

pipenv check
pipenv update
git diff .
git status
git add Pipfile.lock 
git commit -m "Fixes #10 - notebook security."
git push
```

**Comment by clemens-tolboom** [comment]
![github-alert-notebook](https://user-images.githubusercontent.com/371014/55457151-55660100-55e9-11e9-8659-6aeadd37e639.png)
